core/src/main/java/smashdudes/ecs/systems/CrouchingSystem.java

```diff
@@ -0,0 +1,18 @@
+package smashdudes.ecs.systems;
```

Comment:
This system has nothing in it and should be removed

Reply:
Was planning on flushing it out on my next commit

Reply:
Cool. Try to only make the pull request once you have "finished" code